cloudcix_primitives/bridgeifns.py

```diff
@@ -0,0 +1,359 @@
+        3052: f'3052: Failed to connect to the disabled PodNet from the config file {config_file} for payload interface_add:  ',
+        3053: f'3053: Failed to run interface_add payload on the disabled PodNet. Payload exited with status ',
+        3054: f'3054: Failed to connect to the disabled PodNet from the config file {config_file} for payload interface_main:  ',
+        3055: f'3055: Failed to run interface_main payload on the disabled PodNet. Payload exited with status ',
+        3056: f'3056: Failed to connect to the disabled PodNet from the config file {config_file} for payload interface_ns:  ',
+        3057: f'3057: Failed to run interface_ns payload on the disabled PodNet. Payload exited with status ',
+        3058: f'3058: Failed to connect to the disabled PodNet from the config file {config_file} for payload interface_up:  ',
+        3059: f'3059: Failed to run interface_up payload on the disabled PodNet. Payload exited with status ',
+    }
+
+    # Default config_file if it is None
+    if config_file is None:
+        config_file = '/opt/robot/config.json'
+
+    status, config_data, msg = load_pod_config(config_file)
+    if not status:
+        if config_data['raw'] is None:
+            return False, msg
+        else:
+            return False, msg + "\nJSON dump of raw configuration:\n" + json.dumps(config_data['raw'],
+                                                                               indent=2,
+                                                                               sort_keys=True)
+    enabled = config_data['processed']['enabled']
+    disabled = config_data['processed']['disabled']
+
+    def run_podnet(podnet_node, prefix, successful_payloads):
+        rcc = SSHCommsWrapper(comms_ssh, podnet_node, 'robot')
+        fmt = PodnetErrorFormatter(
+            config_file,
+            podnet_node,
+            podnet_node == enabled,
+            {'payload_message': 'STDOUT', 'payload_error': 'STDERR'},
+            successful_payloads
+        )
+
+        payloads = {
+            'interface_check' : f'ip netns exec {namespace} ip link show {namespace}.{bridgename}',
+            'interface_add' : f'ip link add {bridgename}.{namespace} type veth peer name {namespace}.{bridgename}',
+            'interface_main' : f'ip link set dev {bridgename}.{namespace} master {bridgename}',
+            'interface_ns': f'ip link set dev {namespace}.{bridgename} netns {namespace}',
+            'interface_up' : f'ip netns exec {namespace} ip link set dev {namespace}.{bridgename} up',
+        }
+
+        ret = rcc.run(payloads['interface_check'])
+        if ret["channel_code"] != CHANNEL_SUCCESS:
+            return False, fmt.channel_error(ret, f"{prefix+1}: " + messages[prefix+1]), fmt.successful_payloads
+        if ret["payload_code"] == SUCCESS_CODE:
+            ret = rcc.run(payloads['interface_up'])
```

Comment:
Please apply this pull request - https://github.com/CloudCIX/primitives/pull/46

It is what I was referring to in this comment - https://github.com/CloudCIX/primitives/pull/31#discussion_r1784548592